.travis.yml

```diff
@@ -39,5 +42,5 @@ branches:
 script:
   - python3 tools/listapis.py apis/apis.json > /dev/null || travis_terminate 1
   - touch build_timing_log.txt
-  - ./build.sh --diff --regex $API_REGEX
+  - if [ -z "$SPANNER_EMULATOR_HOST" ]; then ./build.sh --diff --regex $API_REGEX; else ./spanner-ITs-against-emulator.sh; fi
```

Comment:
This is where I most need feedback. As a start, I decided not to modify `build.sh` but please let me know if it's better to add an option like `--spanner-ITs` to build.sh and then run this command.

Reply:
Ah I missed that you can configure a separate script per job. Fixed now.